ansible/host_vars/wck.yaml

```diff
@@ -0,0 +1,77 @@
+---
+
+selinux_status:
+  - policy: targeted
+    state: permissive
+
+users:
+  - name: wck
+    group: chlservices
+    gid: 600
+    
+app_user_home_path: /home/wck
+
+s3_file_perl_lib: /application/shared/libraries/perl-xerces-lib.tar.gz
+s3_file_perl_lib_name: '{{ s3_file_perl_lib | basename }}'
+
+tar_files:
+  - "{{ s3_file_perl_lib_name }}"
+
+install_watcher_service: true
+
+postfix_main_config_file: main.cf
+postfix_config:
+  - key: mydomain
+    value: companieshouse.gov.uk
+  - key: myorigin 
+    value: $mydomain
+  - key: relayhost
+    value: smtp-outbound.sharedservices.aws.internal
+
+postfix_net_config:
+  - key: inet_interfaces 
+    value: all
+  - key: inet_protocols 
+    value: ipv4
+
+## Nagios Variables
```

Comment:
Can probably strip out the Nagios code, this is being replaced with CloudWatch going forward.